packages/@o3r/core/schematics/rule-factories/store/index.ts

```diff
@@ -29,25 +28,27 @@ const ngrxRouterStoreDevToolDep = '@ngrx/store-devtools';
  * @param options @see RuleFactory.options
  * @param rootPath @see RuleFactory.rootPath
  * @param options.projectName
- * @param _rootPath
+ * @param projectType
  */
-export function updateStore(options: { projectName: string | null | undefined }, _rootPath: string): Rule {
+export function updateStore(options: { projectName: string | undefined}, projectType?: 'application' | 'library'): Rule {
```

Comment:
the type `'application' | 'library'` can be replace by `WorkspaceProject['projectType']`